docusaurus.config.js

```diff
@@ -9,7 +9,7 @@ const config = {
   tagline: 'Documentation and Community Platform for the Sovereign Cloud Stack',
   url: 'https://docs.scs.community',
   baseUrl: '/',
-  onBrokenLinks: 'throw',
+  onBrokenLinks: 'warn',
```

Comment:
Probably its a good idea to do rigid checking in future.
If the build does not fail, nobody will review the output of the build.